lib/server.js

```diff
@@ -524,12 +536,21 @@ WalletService.prototype.joinWallet = function(opts, cb) {
   if (_.isEmpty(opts.name))
     return cb(new ClientError('Invalid copayer name'));
 
+  opts.derivationStrategy = opts.derivationStrategy || WalletUtils.DERIVATION_STRATEGIES.BIP45;
```

Comment:
Why join needs a `derivationStrategy` param? It seems that should be pulled from the wallet, and if not supported, throw an error.